OpenEdXMobile/res/values/dimens.xml

```diff
@@ -8,6 +8,13 @@
     <!-- Per Material Design Guidelines -->
     <dimen name="action_bar_icon_size">24dp</dimen>
 
+    <!-- Alert dialog default padding -->
+    <dimen name="alert_dialog_padding_width">22dp</dimen>
+    <dimen name="alert_dialog_padding_height">11dp</dimen>
```

Comment:
The default `AlertDialog` message view layout seems to be this: https://github.com/android/platform_frameworks_support/blob/master/v7/appcompat/res/layout/abc_alert_dialog_material.xml#L86

You'll note that it uses the `abc_dialog_padding_top_material` dimen for it's top padding, and the `dialogPreferredPadding` theme attribute for it's left/right padding. Here are their default values if you're interested: https://github.com/android/platform_frameworks_support/blob/master/v7/appcompat/res/values/dimens.xml#L73